image/cli/mascli/functions/pipeline_config_applications

```diff
@@ -217,7 +217,7 @@ function config_pipeline_applications() {
   fi
 
   # Manage
-  if prompt_for_confirm "Install Manage"; then
+  if prompt_for_confirm "Install Manage (SNO supports Manage app)"; then
```

Comment:
Considering that if we detect SNO already, let's display only the options that support SNO instead of adding other options that do not work.

Reply:
I am going to revert the change. Code changes have been made on a different branch where this apps that are not available will not be shown to the user